### PR TITLE
feat: add builtin function 'crypto.uuid()' and 'crypto.filehash()'

### PR DESCRIPTION
**Pull request by zong-zhe**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

re #313 

#### 2. What is the scope of this PR (e.g. component or file name):

feat: add builtin function 'crypto.uuid()' and 'crypto.filehash()'

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [x] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

### CODE REVIEW

File: kclvm/sema/src/builtin/system_module.rs [hunk 1]
```diff
@@ -1155,7 +1155,9 @@ register_json_member! {
 // ------------------------------
 
 pub const CRYPTO: &str = "crypto";
-pub const CRYPTO_FUNCTION_NAMES: &[&str] = &["md5", "sha1", "sha224", "sha256", "sha384", "sha512"];
+pub const CRYPTO_FUNCTION_NAMES: &[&str] = &[
+    "md5", "sha1", "sha224", "sha256", "sha384", "sha512", "uuid",
+];
```
Comment (Peefy) [inline kclvm/sema/src/builtin/system_module.rs:1160]: Lost `uuid` call for `register_crypto_member!`

File: kclvm/sema/src/builtin/system_module.rs [hunk 2]
```diff
@@ -1155,7 +1155,9 @@ register_json_member! {
 // ------------------------------
 
 pub const CRYPTO: &str = "crypto";
-pub const CRYPTO_FUNCTION_NAMES: &[&str] = &["md5", "sha1", "sha224", "sha256", "sha384", "sha512"];
+pub const CRYPTO_FUNCTION_NAMES: &[&str] = &[
```
Comment (Peefy) [inline kclvm/sema/src/builtin/system_module.rs:1158]: Failure of sema/symbol related tests in API and loader.

Comment (Peefy) [reply kclvm/sema/src/builtin/system_module.rs:1158]: > Failure of sema/symbol related tests in API and loader.

Failure of sema/symbol related tests in the loader crate.

Comment (zong-zhe) [reply kclvm/sema/src/builtin/system_module.rs:1158]: fixed

File: kclvm/sema/src/builtin/system_module.rs (hunk 1 above)
Comment (zong-zhe) [reply kclvm/sema/src/builtin/system_module.rs:1167]: fixed

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**